pkg/controller/operconfig/ingressconfig_controller.go

```diff
@@ -0,0 +1,90 @@
+package operconfig
+
+import (
+	"context"
+	"log"
+
+	operv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/names"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+// newIngressConfigReconciler returns a new reconcile.Reconciler
+func newIngressConfigReconciler(client client.Client) *ReconcileIngressConfigs {
+	return &ReconcileIngressConfigs{client: client}
+}
+
+var _ reconcile.Reconciler = &ReconcileIngressConfigs{}
+
+// ReconcileIngressConfigs watches for updates to ingress controller configuration
+// and sets the network policy related labels on the openshift-host-network namespace
+type ReconcileIngressConfigs struct {
+	client client.Client
+}
+
+// Reconcile sets the openshift-host-network namespaces' labels as per the
+// endpointPublishingStrategy of the `default` ingress controller object.
+func (r *ReconcileIngressConfigs) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	if request.Namespace == names.IngressControllerNamespace && request.Name == names.DefaultIngressControllerName {
```

Comment:
return early, don't indent the whole function

    if ! ... {
            return reconcile.Result{}, nil
    }
    log.Printf("Reconciling...